skll/config.py

```diff
@@ -363,6 +365,9 @@ def _parse_config_file(config_path):
             # default number of cross-validation folds
             cv_folds = 10
 
+    # whether or not to save the cv fold ids
+    save_cv_folds = config.get("Input", "save_cv_folds")
```

Comment:
I think this belongs in `Output`, since this an option that relates to what we're outputting, and not to what we're given.